modules/ingest-geoip/build.gradle

```diff
@@ -57,10 +57,8 @@ tasks.named("internalClusterTest").configure {
   }
 }
 
-tasks.named("bundlePlugin").configure {
-  from("${project.buildDir}/ingest-geoip") {
+esplugin.bundleSpec.from("${project.buildDir}/ingest-geoip") {
```

Comment:
Would it make sense to move this up above where we ahve the main `esplugin` config block?

Reply:
fixed